packages/financial-templates-lib/src/proxy-transaction-handler/DSProxyManager.js

```diff
@@ -43,8 +44,19 @@ class DSProxyManager {
     return this.dsProxyAddress;
   }
 
-  // Load in an existing DSProxy for the account EOA if one already exists OR create a new one for the user.
-  async initializeDSProxy() {
+  // Load in an existing DSProxy for the account EOA if one already exists OR create a new one for the user. Note that
+  // the user can provide a dsProxyAddress if they want to override the factory behaviour and load in a DSProxy directly.
+  async initializeDSProxy(dsProxyAddress = null, shouldCreateProxy = true) {
```

Comment:
to enable the script added in this PR, a small bit of extra branching logic was added to let you override some of the default behavour.